src/style.css

```
@@ -0,0 +1,26 @@
+* {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background: url(images/star.svg) center no-repeat;
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
```

```suggestion
.stars__star {
  background: url(images/star.svg) center no-repeat;
  height: 16px;
  width: 16px;
}

.stars__star:not(:last-child) {
  margin-right: 4px;
}
```
I think it's redundant